packages/phone-number-privacy/monitor/src/query.ts

```diff
@@ -0,0 +1,27 @@
+import { newKit, OdisUtils } from '@celo/contractkit'
+import { AuthSigner } from '@celo/contractkit/lib/identity/odis/query'
+import { fetchEnv } from '@celo/phone-number-privacy-common'
+import { normalizeAddressWith0x, privateKeyToAddress } from '@celo/utils/lib/address'
+import * as functions from 'firebase-functions'
+
+const privateKey = fetchEnv('PRIVATE_KEY')
+const phoneNumber = fetchEnv('PHONE_NUMBER')
+const accountAddress = normalizeAddressWith0x(privateKeyToAddress(privateKey)) // 0x1be31a94361a391bbafb2a4ccd704f57dc04d4bb
+
+const contractKit = newKit(functions.config().blockchain.provider)
+contractKit.addAccount(privateKey)
+contractKit.defaultAccount = accountAddress
+
+export const queryOdisForSalt = () => {
+  const authSigner: AuthSigner = {
+    authenticationMethod: OdisUtils.Query.AuthenticationMethod.WALLET_KEY,
+    contractKit,
+  }
+
+  return OdisUtils.PhoneNumberIdentifier.getPhoneNumberIdentifier(
```

Comment:
Can you pass in a client version string with something that indicates this is coming from a canary test and not the mobile client (ex. "canary:1.0.0")? This way we can tell from the Combiner logs that the out-of-quotas are not coming from real users.

Reply:
Good idea